package/MDAnalysis/analysis/__init__.py

```diff
@@ -27,6 +27,20 @@
 
 The :mod:`MDAnalysis.analysis` sub-package contains various recipes and
 algorithms that can be used to analyze MD trajectories.
+If not stated differently, an analysis conducted by the available modules
+always follows the same structure
+
+1. Initialize the object previously imported.
+2. Run the analysis for specific trajectory slices
+3. Access the analysis from the `results` attribute (if available)
+
+ .. code-block:: python
+
+    from MDAnalysis.anlaysis import AnalysisModule
+
+    analysis_obj = AnalysisModule(trajectory, ...)
+    analysis_obj.run(start_frame, stop_frame, step)
+    print(analysis_obj.results)
```

Comment:
@orbeckst I could move some text to package/doc/sphinx/source/documentation_pages/analysis_modules.rst and remove everything else since its doubled and not rendered anywhere in the docs.

Reply:
Yes please, it's a bit strange that this module is so well documented and then appears nowhere.

Reply:
Okay copy everything or just use the beginning of the description. The modules are linked anyway right?